core/probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/Builds.scala

```diff
@@ -109,7 +109,7 @@ object Builds extends IntelliJApi {
         compileContext: CompileContext
     ): Unit = {
       def messages(category: CompilerMessageCategory): Seq[BuildMessage] = {
-        compileContext.getMessages(category).map { msg =>
+        compileContext.getMessages(category).toIndexedSeq.map { msg =>
```

Comment:
why this toIndexedSeq conversion happens everywhere?

Reply:
Without it (as we have on master now) - we are having following warnings (which contain answer, **why**):
```
[warn] /Users/lkontowski/code/ide-probe/core/probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/Builds.scala:112:50: method copyArrayToImmutableIndexedSeq in class LowPriorityImplicits2 is deprecated (since 2.13.0): Implicit conversions from Array to immutable.IndexedSeq are implemented by copying; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call
[warn]         compileContext.getMessages(category).map { msg =>
[warn]                                                  ^
[warn] one warning found
```